experiment/kind-detect-local-e2e.sh

```diff
@@ -54,12 +54,17 @@ signal_handler() {
 }
 trap signal_handler INT TERM
 
-# build kubernetes / node image, e2e binaries
+# build kubernetes / node image, e2e binaries and ginkgo
 build() {
   # build the node image w/ kubernetes
   kind build node-image -v 1
+  # Ginkgo v1 is used by Kubernetes 1.24 and earlier, fallback if v2 is not available.
+  GINKGO_SRC_DIR="github.com/onsi/ginkgo/v2/ginkgo"
+  if [ ! -d "$GINKGO_SRC_DIR" ]; then
```

Comment:
Urgh. You are right, as always.

We can fix this by inverting the logic:
```
GINKGO_SRC_DIR="vendor/github.com/onsi/ginkgo/ginkgo"
if [ ! -d "$GINKGO_SRC_DIR" ]; then
   GINKGO_SRC_DIR="github.com/onsi/ginkgo/v2/ginkgo"
fi
```

This will work also in case that we drop the vendor directory entirely.